www-src/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "scripts": {
     "postinstall": "./node_modules/.bin/brunch build",
-    "test": "/bin/true",
+    "test": "./node_modules/.bin/mocha test/unit/* --reporter spec --compilers coffee:coffee-script/register --colors",
```

Comment:
I think you can omit `./node_modules/.bin/`. This path is automatically added to `$PATH` when using `npm run`.